functional-tests/example.env

```diff
@@ -1,5 +1,6 @@
-BASEURL=http://127.0.0.1:8000/
+BASE_URL=http://127.0.0.1:8000/gwells/
 GWELLS_USERNAME=bdduser1
 GWELLS_VIEWER_USERNAME=bdd-viewer
 GWELLS_SUBMISSION_USERNAME=bdd-submission
-GWELLS_REGISTRY_USERNAME=bdd-registry
\ No newline at end of file
+GWELLS_REGISTRY_USERNAME=bdd-registry
+GWELLS_PASSWORD=
```

Comment:
I included this example.env file, with the intention that you could rename it locally to .env and it would get used.

But maybe it shouldn't be committed at all?  I can always remove it and rebase so the history is wiped if needed.

Reply:
I've been flip flopping on this. I've found it very useful to have .env files ready to go. This may become less of a concern if we all move over to using docker for our environments. Until then I'm ok with having a .env file - but I would drop the GWELLS_PASSWORD, just in case someone specifies something by accident and then commits it.

Reply:
Sure thats fair.  Removed.

I think long term, if we get docker to run the entire app, then this file/logic can be removed.  But in the interim its been useful for me, and hopefully whoever else wants to run the tests.